src/aat/resources/features/common/linked_case/Unauthorised_Case_Creation_Data.td.json

```diff
@@ -0,0 +1,23 @@
+{
+	"_guid_": "Unauthorised_Case_Creation_Data",
```

Comment:
Remove this file and adjust S.140.3 scenario: as there are no unauthorised actions in this call so its naming makes no sense and its uses is not required to still fulfil the scenario.

Reply:
[54f1d4f](https://github.com/hmcts/ccd-data-store-api/pull/1920/commits/54f1d4fcf3faac512d1a6278a71e82b99881cce4)